backend/app/api/endpoints/internal/attachments.py

```
@@ -0,0 +1,60 @@
+from app.services.auth.internal_service_token import verify_internal_service_token
+from app.services.context.context_service import context_service
+
+router = APIRouter(
+    prefix="/attachments",
+    tags=["attachments-internal"],
+    dependencies=[Depends(verify_internal_service_token)],
+)
+
+
+@router.get("/{attachment_id}/download")
+def download_attachment(attachment_id: int, db: Session = Depends(get_db)):
```

Ruff (0.15.12)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)